src/amber/controller/filters.cr

```diff
@@ -18,13 +19,14 @@ module Amber::Controller
         if self.responds_to? :after_filters
           self.after_filters
           @filters.run(:after, action)
+          @filters.run(:after, :except, action)
           @filters.run(:after, :all)
         end
       end
     end
   end
 
-  record Filter, precedence : Symbol, action : Symbol, blk : -> Nil do
+  record Filter, precedence : Symbol, action : Symbol, blk : -> Nil, excepts : Array(Symbol) | Nil = nil do
```

Comment:
What do you think about defaulting the `excepts` array as an empty array and remove the Nil union here.  That way you don't have to do this on line 104.

Reply:
makes sense